test/EstateRegistry.js

```diff
@@ -251,6 +247,19 @@ contract('EstateRegistry', accounts => {
       })
     })
 
+    it('should allow operator to create an estate', async function() {
+      await land.assignMultipleParcels([0], [1], user, sentByCreator)
+      await land.setApprovalForAll(anotherUser, true, sentByUser)
+      await createEstate([0], [1], user, sentByAnotherUser)
+      await assertEstateCount(user, 1)
+    })
+
+    it('fails if sender is not owner or operator of all lands', async function() {
+      await land.assignMultipleParcels([0, 0], [1, 2], user, sentByCreator)
+      await land.approve(anotherUser, 1, sentByUser)
```

Comment:
To approve a parcel shouldn't we use `encodeTokenId` to make it more clear?

Example:

```
      const landId = await land.encodeTokenId(0, 0)
      await land.approve(anotherUser, landId, sentByUser)
```

Reply:
Based on what it is [here](https://github.com/decentraland/land/blob/master/test/EstateRegistry.js#L20) we are not using this function for Estate registry tests

Reply:
Because there is no function in those tests calling a tokenId for LAND.

Reply:
Yes, e.g: https://github.com/decentraland/land/blob/master/test/EstateRegistry.js#L308

Reply:
We should change that to use `encodeTokenId` too. What is 1? I think is difficult to understand what that represents. We have coordinates that maps to tokenId and is clear when we use the mapping function.

Reply:
We need to change it in all places, is not the only one. That's why we put the comment I showed.

I can store it in a const like https://github.com/decentraland/land/blob/master/test/EstateRegistry.js#L322 but I think changing all references to `encondeTokenId`  It is not part of this PR.

Reply:
Ok, let's do it in a different PR for clarity purposes.